vis_server/protocols/FileLoader.py

```diff
@@ -24,7 +24,17 @@ def list_variables(self, file_name):
         out = {}
         for vname in reader.variables:
             var = reader.variables[vname]
+
+            # Get a displayable name for the variable
+            if hasattr(var, 'long_name'):
+                name = var.long_name
+            elif hasattr(var, 'title'):
+                name = var.title
+            elif hasattr(var, 'id'):
+                name = var.id
+
```

Comment:
@jbeezley @chaosphere2112 I think it is worth adding the actual name of the variable id in the file in between parenthesis, user usually know this better than the actual long name if it's their data
e.g.
`"Surface Air Temperature (tas)"`